web/src/Routes.tsx

```diff
@@ -25,12 +25,18 @@ const CustomerExplorePage = lazy(() => import('components/customer/explore'));
 // Merchant Pages
 const MerchantLandingPage = lazy(() => import('components/merchant/landing'));
 const MerchantSignUpPage = lazy(() => import('components/merchant/sign-up'));
+// Shared Pages
```

Comment:
can you add more comment about how is it shared?

Reply:
@ifhuang Going to a particular listing page(`https://gpay-group-buy.an.r.appspot.com/listing/:listingId`) is a common route for both merchants and customers. But our app will show the user to the correct component depending on whether they are a merchant or not.

Reply:
yes. add some context to the comment in the codebase

Reply:
@ifhuang So just something like this?
```suggestion
// Shared Pages (neither merchant nor customer specific)
```

Reply:
how about:
Shared Pages. Different components may be displayed depending on whether user is a customer or merchant

Reply:
Leaving the conversation we had on Chat here:

> Caryn Heng, 21 mins
I feel like it's not only pages like that that are considered Shared.
I only did the `// Customer Pages` and `// Merchant Pages` comments to organise how the imports look
and `// Shared Pages` is just pages that kind of belong to neither HAHA
Idk if the router (which only cares about routing url path to components) should be having comments such as "Different components may be displayed depending on whether user is a customer or merchant"
that feels like a comment that should be left for the ListingDetails shared component itself

> Caryn Heng, 12 mins
or does `// Common Pages` sound better? haha
I was thinking that other pages that are under the `// Shared Pages` comment can be pages whereby both of them see the exact same thing
1 simple example is the 404 page
And if I added the "Different components may be displayed depending on whether user is a customer or merchant" part the 404 page would not fit in there anymore

> Yifu Huang, 7 mins
ic
got your idea now. yeah then probably "Common Pages" sounds better


Will proceed to rename the comment to "Common Pages"!